src/main/resources/reference.conf

```diff
@@ -84,6 +84,11 @@ lorre {
   GETResponseEntityTimeout: 90 s
   POSTResponseEntityTimeout: 1 s
 
+  # defines custom updates to be run at specific epochs (i.e. upon reaching a given level)
+  chain-epoch-updates {
```

Comment:
`epoch` is a dangerous word to use as it has specific meanings in blockchain protocols. Could we please remove all instances of it in these code changes? Over here, I would suggest something like:

```
account-refreshes {
  account-refresh-levels: []
}
```

I would recommend renaming the table to something like `account_refresh_levels`.

Reply:
Ouch, I didn't know. I was trying to find some significant naming to identify a specific point in time... I'll work on that